packages/console/src/ds-components/Select/index.tsx

```diff
@@ -58,6 +67,55 @@ function Select<T extends string>({
     event.stopPropagation();
   };
 
+  const getSearchInputContainerStyles = () => {
+    if (!anchorRef.current) {
+      return {};
+    }
+    const element = anchorRef.current;
+    const cs = getComputedStyle(element);
+    const paddingX = Number.parseFloat(cs.paddingLeft) + Number.parseFloat(cs.paddingRight);
+    const paddingY = Number.parseFloat(cs.paddingTop) + Number.parseFloat(cs.paddingBottom);
+    const borderX = Number.parseFloat(cs.borderLeftWidth) + Number.parseFloat(cs.borderRightWidth);
+    const borderY = Number.parseFloat(cs.borderTopWidth) + Number.parseFloat(cs.borderBottomWidth);
+    return {
+      position: 'fixed',
+      width: `${element.offsetWidth - paddingX - borderX}px`,
+      height: `${element.offsetHeight - paddingY - borderY}px`,
+      top: `${
+        element.getBoundingClientRect().top +
+        Number.parseFloat(cs.borderTopWidth) +
+        Number.parseFloat(cs.paddingTop)
+      }px`,
+      left: `${
+        element.getBoundingClientRect().left +
+        Number.parseFloat(cs.borderLeftWidth) +
+        Number.parseFloat(cs.paddingLeft)
+      }px`,
+      backgroundColor: cs.backgroundColor,
+    };
+  };
+
+  useWindowResize(() => {
+    setIsOpen(false);
+  });
```

Comment:
Is this expected? Closing the dropdown on window resize?

Reply:
@charIeszhao I thought it would be much easier and simpler approach.